eas/api/views.py

```diff
@@ -309,7 +310,8 @@ def paypal_accept(request):
 
 @api_view(["GET"])
 def instagram_preview(request):
-    post_url = request.GET["post_url"]
+    encoded_post_url = request.GET["post_url"]
+    post_url = urllib.parse.unquote(encoded_post_url)
```

Comment:
If the URL comes in as a QS param, we need to encode it

Reply:
I can also change this to a POST if it make things easier.